components/eam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -890,12 +894,12 @@ subroutine micro_p3_tend(state, ptend, dtime, pbuf)
     call pbuf_get_field(pbuf,    snow_sed_idx,  snow_sed)
     call pbuf_get_field(pbuf,    prec_pcw_idx,  prec_pcw)
     call pbuf_get_field(pbuf,    snow_pcw_idx,  snow_pcw)
-    !============================ 
+    !============================
     ! All internal PBUF variables
     ! INPUTS
     call pbuf_get_field(pbuf,      relvar_idx,    relvar                                                   )
     call pbuf_get_field(pbuf,      t_prev_idx,    t_prev                                                   )
-    call pbuf_get_field(pbuf,     qv_prev_idx,    qv_prev                                                  )
+    call pbuf_get_field(pbuf,     qv_prev_idx,    qv_prev_wet                                                  )
```

Comment:
Here I just changed the name of the PBUF variable to explicitly mention that it is a `wet` mmr. I am using qv_prev_dry internally in the P3 code.

Reply:
I think perhaps what Luca is asking is whether we don't just want to store qv_prev_dry in the field manager directly since (if I recall correctly, no other process touches it. The benefit would be saving the cost of a conversion to/from wet, but the cost would be violation of our "FM only stores wet mixing ratios" rule, which could be a dangerous precedent. My inclination is to vote for storing qv_prev_wet (as you do now).